examples/variational_autoencoder.py

```diff
@@ -39,7 +41,7 @@ def sampling(args):
 
 
 def vae_loss(x, x_decoded_mean):
-    xent_loss = original_dim * objectives.binary_crossentropy(x, x_decoded_mean)
+    xent_loss = objectives.binary_crossentropy(x, x_decoded_mean)
```

Comment:
Are we sure about this removal of the multiplication by the original_dim?
@RuiShu added this multiplication in #3220, and he had some arguments to do it.
Initial implementation was without this multiplication.